msteams-platform/toolkit/create-resource-bot-microsoft-entra-id.md

```diff
@@ -0,0 +1,193 @@
+
+    The **Overview** page opens as the deployment is in progress. Once the deployment is complete, you get a **Deployment succeeded** message. This means that the bot resource is successfully created and the selected resources are provisioned.
+
+1. Under **Next steps** dropdown, select **Go to resource**.
+
+    :::image type="content" source="../assets/images/authentication/go-to-bot-resource.png" alt-text="Screenshot shows the Azure bot resource deployment page.":::
+
+     You can see the details of the bot resource you deployed.
+
+    :::image type="content" source="../assets/images/authentication/bot-resource-overview-page.png" alt-text="Screenshot shows the overview page of the resource bot":::
+
+After you create your bot resource, you need to add a client secret and enable the bot to work in Teams.
+
+### Create client secret
+
+A client secret is a string that the bot app uses to prove its identity when requesting a token from Microsoft Entra ID.
+
+#### To create client secret for the bot
+
+1. Open the [Azure portal](https://ms.portal.azure.com/) on your web browser.
+
+1. Enter the name of your bot resource in **Search** box and open your bot.
+
+1. Select **Configuration**.
+
+    :::image type="content" source="../assets/images/authentication/resource-bot-configuration-select.png" alt-text="Screenshot show the resource bot overview page with the configuration option highlighted in red.":::
+
+    The **Configuration** page appears.
+
+1. Under **Microsoft App ID**, select **Manage Password**.
+
+    :::image type="content" source="../assets/images/authentication/bot-config-manage-password.png" alt-text="Screenshot shows the manage password option under Microsoft app ID.":::
+
+     The **Certificates & secrets** page appears.
+
+2. Select **+ New client secret**.
+
+    :::image type="content" source="../assets/images/authentication/client-secret.png" alt-text="Screenshot shows the add a client secret option highlighted in red.":::
+
+   The **Add a client secret** page appears.
+
+    :::image type="content" source="../assets/images/authentication/add-a-client-secret.png" alt-text="Screenshot shows the add a client secret page.":::
+
+3. Enter the description, select the duration of validity for the secret, and select **Add**.
+
+    A message pops up on the browser stating that the credentials of the bot resource were updated. The client secret displays on the page.
+
+    :::image type="content" source="../assets/images/authentication/client-secret-updated.png" alt-text="Screenshot shows the client secret of the bot resource.":::
+
+> [!NOTE]
+> Ensure that you copy the value of client secret right after you create it. The value is visible only at the time when the client secret is created, and can't be viewed after that.
+
+### Enable bot for Teams
+
+You must enable the Teams channel to let the bot interact with Microsoft Teams.
```

Comment:
We'll need to clarity here as to why we need to enable the Teams channel, not sure what this sentence achieves currently.